README.md

```diff
@@ -329,17 +329,29 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
   > **NOTE** ⏳ This command can take few minutes or immediately depending on your internet connection or model is cached. If you have issues connecting to Hugging Face, refer to the [Hugging Face discussion forum](https://discuss.huggingface.co/) for more details.
 
+  #### Downloading another instructLab model from Hugging Face
+
+  If you wish to download another [instructlab model](https://huggingface.co/instructlab) from Hugging Face, for example the [instructlab/granite-7b-lab-GGUF](https://huggingface.co/instructlab/granite-7b-lab-GGUF) model, you can  use this command with additional parameters as follows:
```

Comment:
```suggestion
  If you wish to download another [InstructLab model](https://huggingface.co/instructlab) from Hugging Face, for example the [`instructlab/granite-7b-lab-GGUF`](https://huggingface.co/instructlab/granite-7b-lab-GGUF) model, you can  use this command with additional parameters as follows:
```

Reply:
@bjhargrave Thanks BJ, like the suggestion.